ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -957,7 +957,7 @@ spec:
         - name: REQUEST_URL
           value: "$(params.git_pr_url)"
         - name: COMMENT
-          value: "[View your certification test results and logs here]($(tasks.set-env.results.connect_url)/projects/$(tasks.certification-project-check.results.certification_project_id)/test-results)."
+          value: "[View your certification test results and logs here]($(tasks.set-env.results.connect_url)/internal/technology-partner-data-hub/#/projects/$(tasks.certification-project-check.results.certification_project_id)/test-results)."
```

Comment:
This is the internal path, partners do not have access to this. We show the URL that they can access.

Reply:
I agree, the link is primarily for partners and it needs to be accessible from the internet.